specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/ga/2021-03-07/phonenumbers.json

```diff
@@ -0,0 +1,932 @@
+            "Inbound",
+            "Outbound",
+            "InboundAndOutbound"
+          ],
+          "type": "string"
+        },
+        "sms": {
+          "description": "Capability value for SMS",
+          "enum": [
+            "None",
+            "Inbound",
+            "Outbound",
+            "InboundAndOutbound"
+          ],
+          "type": "string"
+        }
+      }
+    },
+    "PhoneNumberCost": {
+      "description": "The incurred cost for a single phone number",
+      "type": "object",
+      "properties": {
+        "amount": {
+          "format": "double",
+          "description": "The cost amount.",
+          "type": "number"
+        },
+        "currencyCode": {
+          "description": "The ISO 4217 currency code for the cost amount. e.g. USD",
+          "type": "string"
+        },
+        "billingFrequency": {
+          "description": "The frequency with which the cost gets billed.",
+          "enum": [
+            "Monthly"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "BillingFrequency",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "AcquiredPhoneNumber": {
+      "description": "Represents an acquired phone number.",
+      "required": [
+        "assignmentType",
+        "capabilities",
+        "countryCode",
+        "id",
+        "phoneNumber",
+        "phoneNumberType"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The id of the phone number",
+          "type": "string"
+        },
+        "phoneNumber": {
+          "description": "String of the E.164 format of the phone number",
+          "type": "string"
+        },
+        "countryCode": {
+          "description": "The ISO 3166-2 code of the phone number's country",
+          "type": "string"
+        },
+        "phoneNumberType": {
+          "description": "The phone number's type",
+          "enum": [
+            "Geographic",
+            "TollFree",
+            "Unknown"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberType",
+            "modelAsString": true
+          }
+        },
+        "capabilities": {
+          "$ref": "#/definitions/PhoneNumberCapabilities"
+        },
+        "assignmentType": {
+          "description": "The assignment type of the phone number.",
+          "enum": [
+            "Unknown",
+            "User",
+            "Application"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberAssignmentType",
+            "modelAsString": true
+          }
+        },
+        "purchaseDate": {
+          "format": "date-time",
+          "description": "The date that the phone number is purchased.",
+          "type": "string"
+        },
+        "callbackUrl": {
+          "description": "The callback URL the phone number is assigned to.",
```

Comment:
"description": "The webhook for receiving incoming events.",

Reply:
Done.